controllers/volumereplicationgroup_controller.go

```diff
@@ -78,13 +78,13 @@ func (r *VolumeReplicationGroupReconciler) SetupWithManager(mgr ctrl.Manager) er
 		Complete(r)
 }
 
+// predicate functions send reconcile requests for create and delete events.
```

Comment:
Nice, moving comment out of code.

(nit-style) Add the function name to the beginning of the comment

Reply:
ACK